app/models/developers/celebration_package_request.rb

```diff
@@ -1,8 +1,8 @@
-module Hired
-  class Form < ApplicationRecord
-    include Hired::Notifications
+module Developers
+  class CelebrationPackageRequest < ApplicationRecord
+    include Developers::CelebrationPackageRequests::Notifications
 
-    self.table_name = "hired_forms"
+    self.table_name = "developers_celebration_package_requests"
```

Comment:
Can we move this to the `Developers` module? See `app/models/hiring_agreements.rb:3`.

Reply:
Moved in 602f8c43649a7d7dbea5a19c7a1e9cc2691cc5ad